### PR TITLE
[11.0][ADD] project_parent_task_filter

### PR DESCRIPTION
**Pull request by emagdalenaC2i**
This module adds a filter to show only the parent tasks in a project

### CODE REVIEW

**Review by elicoidal — APPROVED** [review]
LGTM (not yet tested)

**Comment by emagdalenaC2i** [comment]
Any clue about the failures of Runbot and Travis is appreciated?
In Travis: 
_Couldn't load module project_parent_task_filter
cannot import name 'project'_

**Comment by pedrobaeza** [comment]
You have missed to add project.py file.

File: project_parent_task_filter/__init__.py [hunk 1]
```diff
@@ -0,0 +1 @@
+from . import models
```
Comment (carlosdauden) [inline project_parent_task_filter/__init__.py]: Is needed?

**Review by carlosdauden — CHANGES_REQUESTED** [review]

File: project_parent_task_filter/__init__.py (hunk 1 above)
Comment (emagdalenaC2i) [reply project_parent_task_filter/__init__.py]: No. I was looking that right now, I could delete the models folder.
And the project_parent_task_filter/__init__.py should be empty or could be deleted?

Comment (carlosdauden) [reply project_parent_task_filter/__init__.py]: Remove model folder and empty project_parent_task_filter/init.py but don't remove it.

**Review by carlosdauden — APPROVED** [review]

**Comment by dreispt** [comment]
This seems too small to be a module.
Can we add this to an existing module instead?

**Comment by emagdalenaC2i** [comment]
You are right, but I don´t know which module fits better with it. Maybe none.
Also I am working in an improvement that will add more functionality.

**Pull request closed without merging**